### DISCUSSION TITLE
Is there a way to not commit the changes when updating?

### DISCUSSION THREAD

**Discussion by aucampia** [copier-org/copier#880] [Q&A]
I would like to use VSCode to review and integrate changes, if copier just updated the files and staged the changes, or really even just left it as changes (like cruft does) it would be easier for me to then use VScode to review them. Is there such an option?

**Comment by mshafer-NI** [comment]
VSCode has great Git integrations. If you're copying into a Git repo, VSCode can highlight the changed files and show any diffs from before to now.

**Reply by aucampia** [reply]
Thanks, actuall I think I just misunderstood how copier works and I think it is already doing what I wanted it to do, and even more so when I use `--conflict inline`.

**Reply by yajo** [reply]
Yes, it's a recent feature still experimental, but that's what you're looking for.
